### PR TITLE
Silence Now Affects Orders and Plasma/Cooldown Cost Reduction

### PR DESCRIPTION
**Pull request by Surrealaser**
## About The Pull Request

1. Muted marines can no longer issue orders; a Muted marine will have his ongoing orders cancelled next tick.
2. Marines subject to Silence can no longer benefit from orders.
3. Silence plasma cost reduced from 75 to 50.
4. Silence cooldown reduced from 40 to 30 seconds.
5. Haunt removed.
6. Xenos can now see which marines are subject to Silence via a HUD element.
7. Silence duration fixed; it is now actually 10 seconds.
8. Silence windup now 0.3 seconds reduced from 0.5 seconds.

## Why It's Good For The Game

Muting and deafness now have interactions with and bearing on Orders that they should have had since the beginning.

Also makes Silence not as prohibitively costly as it was.

## Changelog
:cl:
balance: Muted marines can no longer issue orders; a Muted marine will have his ongoing orders cancelled next tick.
fix: Muted marines can no longer issue orders.
balance: Marines subject to Silence can no longer benefit from orders.
fix: Marines subject to Silence can no longer benefit from orders.
balance: Silence plasma cost reduced from 75 to 50.
balance: Silence cooldown reduced from 40 to 30 seconds.
balance: Silence windup reduced from 0.5 seconds to 0.3 seconds.
qol: Xenos can now see which marines are subject to Silence via a HUD element.
fix: Silence duration fixed; it is now actually 10 seconds.
remove: Haunt removed.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/human/life/handle_status_effects.dm [hunk 1]
```diff
@@ -17,10 +17,10 @@
 	if(command_aura)
 		command_aura_tick--
 
-		if(command_aura_tick < 1)
+		if(command_aura_tick < 1 || IsMute()) //Null the command aura if we're muted or its duration is over
 			command_aura = null
 
-		if(stat == CONSCIOUS)
+		if(stat == CONSCIOUS && !isdeaf()) //Must be conscious and not deaf.
```
Comment (TiviPlus) [inline code/modules/mob/living/carbon/human/life/handle_status_effects.dm:23]: Remove this change this is a severe orders nerf

**Review by TiviPlus — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/human/life/handle_status_effects.dm (hunk 1 above)
Comment (Surrealaser) [reply code/modules/mob/living/carbon/human/life/handle_status_effects.dm]: Changed this to require the marine be both deafened and muted, so only the Silence ability can cancel orders.

File: code/modules/mob/living/carbon/xenomorph/castes/hunter/abilities_hunter.dm [hunk 2]
```diff
@@ -480,15 +435,16 @@
...
 		var/silence_multiplier = 1
 		if(mark_target == target) //Double debuff stacks for the marked target
 			silence_multiplier = HUNTER_SILENCE_MULTIPLIER
 		to_chat(target, span_danger("Your mind convulses at the touch of something ominous as the world seems to blur, your voice dies in your throat, and everything falls silent!") ) //Notify privately
 		target.playsound_local(target, 'sound/effects/ghost.ogg', 25, 0, 1)
 		target.adjust_stagger(HUNTER_SILENCE_STAGGER_STACKS * silence_multiplier)
 		target.adjust_blurriness(HUNTER_SILENCE_SENSORY_STACKS * silence_multiplier)
-		target.adjust_ear_damage(deaf = HUNTER_SILENCE_SENSORY_STACKS * silence_multiplier)
-		target.apply_status_effect(/datum/status_effect/mute, HUNTER_SILENCE_DURATION * silence_multiplier)
+		target.adjust_ear_damage(HUNTER_SILENCE_SENSORY_STACKS * silence_multiplier, HUNTER_SILENCE_SENSORY_STACKS * silence_multiplier) //Ear damage + temp deafness
```
Comment (TiviPlus) [inline code/modules/mob/living/carbon/xenomorph/castes/hunter/abilities_hunter.dm:446]:
```suggestion
		target.adjust_ear_damage(HUNTER_SILENCE_SENSORY_STACKS * silence_multiplier, HUNTER_SILENCE_SENSORY_STACKS * silence_multiplier)
```

File: code/game/data_huds.dm [hunk 3]
```diff
@@ -214,6 +216,15 @@
 
 	hud_list[XENO_REAGENT_HUD] = xeno_reagent
 
+	//Xeno debuff section start
+	xeno_debuff.overlays.Cut()
+	xeno_debuff.icon_state = ""
+	if(stat != DEAD)
+		if(IsMute() && isdeaf(src)) //Apply Silence debuff image
+			xeno_debuff.overlays += hunter_silence_image
+
```
Comment (TiviPlus) [inline code/game/data_huds.dm:225]:
```suggestion
		if(IsMute())
			xeno_debuff.overlays += hunter_silence_image

```

**Review by TiviPlus — CHANGES_REQUESTED** [review]

**Review by TiviPlus — APPROVED** [review]

**Review by BraveMole — APPROVED** [review]

**Pull request merged**